Framework/Algorithms/test/ResampleXTest.h

```diff
@@ -161,6 +170,258 @@ class ResampleXTest : public CxxTest::TestSuite {
     // Remove workspace from the data service.
     AnalysisDataService::Instance().remove(outWSName);
   }
+
+  // Set-up a generic function for running tests for EventWorkspace
+  void do_testResampleXEventWorkspace(EventType eventType, bool inPlace,
+                                      bool PreserveEvents,
+                                      bool expectOutputEvent) {
```

Comment:
I don't see why `expectOutputEvent` is a separate flag? Isn't it always the same as `PreserveEvents`?